sdk/eventhub/azure-eventhub/azure/eventhub/_transport/_base.py

```diff
@@ -0,0 +1,223 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
this file might not be needed anymore. Just added it originally to keep track of methods I'd need for both pyamqp and uamqp.